Base/usr/share/man/man5/ipc.md

```diff
@@ -1,17 +1,84 @@
 ## Name
 
-ipc - Inter Process Communication endpoint definition format
+IPC - Inter-Process Communication endpoint definition format (.ipc)
 
 ## Synopsis
 
 The IPC format of SerenityOS is a domain-specific language (DSL) used to define communication endpoints for IPC.
 
 ## Description
 
-These files are human-readable, have no easily detectable filemagic, and define IPC interfaces.
-The format is loosely inspired by C++ headers.
+Informally, IPC files - with the help of the IPC compiler - are used to generate message classes that will **wrap**
```

Comment:
why is **wrap** bold?

Reply:
I wanted to emphasize this, but now looking at it, I don't think it's anything special.